### ISSUE TITLE
media-gfx/inkscape-0.92.3-r1 failure

### ISSUE THREAD

**Issue by wolfwood** [InBetweenNames/gentooLTO#166]
ege-color-prof-tracker.cpp:54:10: fatal error: helper/sp-marshal.h: No such file or directory
 #include "helper/sp-marshal.h"

main difference in my setup is that I'm using gold linker

[inkscape-build.log](https://github.com/InBetweenNames/gentooLTO/files/2543474/inkscape-build.log)

[emerge-info.txt](https://github.com/InBetweenNames/gentooLTO/files/2543477/emerge-info.txt)

**Comment by wolfwood** [comment]
I've successfull compiled this version twice in the past and 0.92.3 5' times. but I've also seen this failure multiple times.

**Comment by InBetweenNames** [comment]
Yeah this is strange.  We'll leave this issue up in case anyone else can reproduce it.

**Comment by wolfwood** [comment]
`media-gfx/inkscape CMAKE_MAKEFILE_GENERATOR=emake` seems to resolve this issue

**Comment by ionenwks** [comment]
I haven't reproduced it but (from a quick look) since `helper/sp-marshal.h` is a generated file, I'd believe there's dependencies issue with the make scripts and `-j 36 -l 36` that you use alters the order things get build in based on load average leading to sometimes the right order, and sometimes not. I don't think this has anything to do with the build flags from LTO (at most it may produce different results due to certain tasks taking longer and being more intensive altering the load average).

Maybe try if still get the issue with a more modest `-j` and no `-l`. Or without `-j` at all.

**Comment by wolfwood** [comment]
yeah, I'm rebuilding with `MAKEOPTS=-j1` i thought the race condition was a
bug in the ninja backend, but I guess it's a bug in the cmake scripts.
